website/docs/cli.md

```diff
@@ -3,8 +3,83 @@ id: cli
 title: CLI
 ---
 
-TODO: Add CLI reference here.
+Docusaurus provides a set of scripts to help you generate, serve, and deploy your website. 
 
-#### References
+Once your website is generated, your website package will contain the Docusaurus scripts that you may invoke with your package manager:
 
-- TODO
+```json
+// package.json
+{
+  // ...
+  "scripts": {
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy"
+  }
+}
+```
+
+## Docusaurus CLI commands
+
+Below is a list of Docusaurus CLI commands and their usages:
+
+### `docusaurus init`
+
+The `docusaurus init` command is intended to be used with `docusaurus` installed globally:
+
+```shell
+$ yarn global add docusaurus
+# or
+$ npm install --global docusaurus
+```
+
+<!-- TODO: init docs after the init command is implemented -->
+
+### `docusaurus start`
+
+Builds and serves the static site with [Webpack Dev Server](https://webpack.js.org/configuration/dev-server).
+
+**options**
+
+|Options|Default|Description|
+|-|-|-|
+|`--port`|`3000`|Specifies the port of the dev server|
+|`--host`|`localhost`|Specifie a host to use. E.g., if you want your server to be accessible externally, you can use `--host 0.0.0.0`|
+|`--no-watch`|`false`|<!-- TODO verify how this works --> Disables watching files for hot reload|
```

Comment:
The `no-watch` is still buggy. Need to investigate

Reply:
OK, let's still leave this line of doc here, will revisit once that is fixed.